README.md

```diff
@@ -14,102 +14,110 @@ library can help with that!
 To get started, you will need a [GitHub Personal Auth
 Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/)
 to make requests to the GitHub API. This should be set in your environment or
-shell ``rc`` file with the name ``GITHUB_API_TOKEN``:
```

Comment:
Where is this markdown formatting coming from? Are you using a tool for doing this linting?

Reply:
Yes, I've been using most of the recommendations from [vscode-markdownlint](https://github.com/DavidAnson/vscode-markdownlint).

Reply:
any way to do this in travis?

Reply:
I'm not particularly well versed in Travis yet. However, I know that the VS Code extension is powered by markdownlint for Node.js. If we can use the npm package in Travis, we'll have the equivalent markdownlint command line tool.